packages/traefik/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,15 +0,0 @@
---- charts-original/Chart.yaml
-+++ charts/Chart.yaml
-@@ -1,10 +1,7 @@
- annotations:
--  artifacthub.io/changes: "- \"⬆️ Upgrade traefik Docker tag to v2.9.8\"\n- \"\U0001F6A8
--    Fail when enabling PSP on Kubernetes v1.25+ (#801)\"\n- \"Separate UDP hostPort
--    for HTTP/3\"\n- \"feat: Add volumeName to PersistentVolumeClaim (#792)\"\n- \"Allow
--    setting TLS options on dashboard IngressRoute\"\n"
-+  fleet.cattle.io/bundle-id: k3s
```

Comment:
Need to keep this change to Chart.yaml in order to make the Rancher UI not offer to upgrade the chart.